contracts/modules/delegates/IexecERC20DelegateKYC.sol

```diff
@@ -1,39 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
-
-/******************************************************************************
- * Copyright 2020 IEXEC BLOCKCHAIN TECH                                       *
- *                                                                            *
- * Licensed under the Apache License, Version 2.0 (the "License");            *
- * you may not use this file except in compliance with the License.           *
- * You may obtain a copy of the License at                                    *
- *                                                                            *
- *     http://www.apache.org/licenses/LICENSE-2.0                             *
- *                                                                            *
- * Unless required by applicable law or agreed to in writing, software        *
- * distributed under the License is distributed on an "AS IS" BASIS,          *
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
- * See the License for the specific language governing permissions and        *
- * limitations under the License.                                             *
- ******************************************************************************/
-
-pragma solidity ^0.6.0;
-pragma experimental ABIEncoderV2;
-
-import "./IexecERC20Delegate.sol";
-import "./IexecERC20CoreKYC.sol";
-
-
-contract IexecERC20DelegateKYC is IexecERC20Delegate, IexecERC20CoreKYC
-{
-	function _beforeTokenTransfer(address from, address to, uint256 amount)
```

Comment:
This could also be removed from the `ERC20Core` contract https://github.com/iExecBlockchainComputing/PoCo/blob/904c4abd942fc2cad6fd9868aabc37db377a3c10/contracts/modules/delegates/IexecERC20Core.sol#L38